Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM odise/busybox-node
```

Comment:
The above mentioned [dockerfile/nodejs](https://dockerfile.github.io/#/nodejs). It's also listed in the previously linked page.
I have an WIP Dockerfile based on my own images, but I preferred not sending it as PR to be used as the 'official' Doorman project Dockerfile, because it's based in my own, opnionated, best practices that not everyone might share. So, instead of sending mine, I was thinking in sending a dockerfile/nodejs based one. Since all images are somewhat opinionated, I see more appropiate to use a NodeJS community-maintained one, which captures that community's best practices and opinions.